### DISCUSSION TITLE
Uninstall Multisite

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7383] [Troubleshooting]
Is there any docs on how to remove a multisite? I have just removed the unrequired sites from config>statamic>sites.php to default me back to a single site but i'm getting `Call to a member function direction() on null (View: /var/www/html/vendor/statamic/cms/resources/views/layout.blade.php) (View: /var/www/html/vendor/statamic/cms/resources/views/layout.blade.php)` as the application is still looking for the required views

**Comment by edalzell** [comment]
Ya you'll need to move the files around as well, much of the content gets nested under the site handles.

Can you roll back the commit where you enabled multi-site?

**Reply by ottz0** [reply]
No I merged this back many branches ago

**Reply by edalzell** [reply]
Sure but there will be a commit that shows what files where moved where, right?

You have to either move the files back so they're not nested under the site handle (like in `collections`), or start again and copy all the content/templates into a blank site.
